packages/api-key-stamper/src/utils.ts

```diff
@@ -9,12 +15,39 @@ export function convertTurnkeyApiKeyToJwk(input: {
 
   const jwk = pointDecode(hexStringToUint8Array(compressedPublicKeyHex));
 
-  jwk.d = hexStringToBase64urlString(uncompressedPrivateKeyHex);
+  // First make a copy to manipulate
+  const jwkCopy = { ...jwk };
+
+  // Ensure that each of the constituent parts are sufficiently padded
+  const paddedD = hexStringToBase64urlString(
+    uncompressedPrivateKeyHex,
+    DEFAULT_JWK_MEMBER_BYTE_LENGTH
+  );
+
+  // Manipulate x and y
+  const decodedX = base64urlToBuffer(jwkCopy.x!);
+  const paddedX = hexStringToBase64urlString(
+    uint8ArrayToHexString(new Uint8Array(decodedX)),
+    DEFAULT_JWK_MEMBER_BYTE_LENGTH
+  );
 
-  return jwk;
+  const decodedY = base64urlToBuffer(jwkCopy.y!);
+  const paddedY = hexStringToBase64urlString(
+    uint8ArrayToHexString(new Uint8Array(decodedY)),
+    DEFAULT_JWK_MEMBER_BYTE_LENGTH
+  );
+
+  jwkCopy.d = paddedD;
+  jwkCopy.x = paddedX;
+  jwkCopy.y = paddedY;
```

Comment:
What led to padding (x, y, d) as opposed to just `d`? Given the jwk is already created I think we're good with the existing components within it (x, y)